demo/demo_walkthrough.md

```diff
@@ -0,0 +1,237 @@
+
+![Task details window](docs/images/task-details.gif)
+
+#### 9b. Task details in Data Explorer
+
+The Data Explorer tab allows you to view the log files and output files generated for each task in its' working directory within the Platform.
+
+You can view, download, and retrieve the link for these intermediate files from the Explorer tab.
+
+![Task data explorer](docs/images/sp-cloud-task-data-explorer.gif)
+
+### 10. Resuming a Run
+
+Seqera Platform enables you to use Nextflow's resume functionality to resume a workflow run with the same parameters, using the cached results of previously completed tasks and only executing failed and pending tasks. If a Run fails or is Cancelled, click on the three dots next to the Run and select 'Resume' from the options menu. You will have the option to edit some parameters before launch. Unlike a relaunch, you cannot edit the pipeline to launch or the work directory during a run resume.
+
+![Resuming a run](docs/images/sp-cloud-resume-a-run.gif)
+
+### 11. Task Status and Cached Processes
+
+The Runs page for a workflow will display the status of tasks in real time as they progress from 'Submitted' to 'Running' to 'Succeeded' or 'Failed'.
+
+If you are resuming a run that had tasks that were completed successfully, you will see a number of tasks shown as 'Cached'.
+
+![Cached processes](docs/images/sp-cloud-cached-processes.gif)
+
+### 12. View Run outputs in Data Explorer
+
+With Data Explorer, you can browse and interact with remote data storage. Data Explorer can be used to view the outputs of your pipelines.
+
+From the View cloud bucket page, you can:
+
+1. Preview and download files: Select the download icon in the 'Actions' column to download a file directly from the list view. Select a file to open a preview window that includes a Download button.
+2. Copy bucket/object paths: Select the Path of an object on the cloud bucket page to copy its absolute path to the clipboard. Use these object paths to specify input data locations during pipeline launch, or add them to a dataset for pipeline input.
+
+![Data Explorer bucket](docs/images/sp-cloud-data-explorer.gif)
+
+### 13. Optimize the Pipeline
+
+When a run completes successfully, an optimized profile is created. This profile consists of Nextflow configuration settings for each process and each resource directive (where applicable): cpus, memory, and time. The optimized setting for a given process and resource directive is based on the maximum use of that resource across all tasks in that process.
+
+Navigate back to the Launchpad, click on the nf-core/rnaseq Pipeline added, and click on the 'Lightbulb' icon to view the optimized profile.
+
+![Optimized configuration](docs/images/optimize-configuration.gif)
+
+**Congratulations!**
+You just added a pipeline to the Launchpad, and executed it on Seqera Platform! 🚀
```

Comment:
```suggestion
You just added a pipeline to the Launchpad, and executed it on Seqera Platform! 🚀

###  14. Credentials (optional for demo)

To execute each run, a job is submitted to your executor. Be it cloud or HPC, the Seqera Platform will require some credentials to submit the job on your behalf. An Admin user can add these keys to the "Credentials" on behalf of the other workspace users. The "Credentials" tab can also be used to add keys and tokens for container registries or Git hosting providers. Our demonstration so far has pulled public data and public workflows, but the Seqera Platform can just as easily be used to execute private workflows using private containers.


### 15. Compute Environments (optional for demo)

As introduced earlier - the Seqera Platform does not host data nor perform the computation - it's a lens into your own infrastructure where the data and computation always remain. Before any runs can be submitted in a workspace, an Admin user needs to add one or more Compute Environments to the workspace. The Compute Environment creation has two paths - a "manual" mode to add pre-existing infrastructure to the workspace, and a "forge" mode where the Seqera Platform itself creates the infra in your cloud account necessary to run Nextflow workloads.

If you have an experienced cloud team, or systems administrators who have already built and administer your cloud or HPC infrastructure, then you only need provide the queue names to which the jobs can be submitted. If you would prefer the platform to set up the infrastructure on your behalf, the user is provided more options to customize the final compute environment. 

Add a CE is done once for the workspace. It takes only a few minutes and is then available for all users in the workspace as an environment in which they can execute their runs.
```

Reply:
I agree, I tend to highlight these things too (time dependent). I was going to add the CE creation, credentials, overview of secrets as separate markdown docs (the main one is getting quite lengthy) - thoughts?